packages/nextra/src/server/rehype-plugins/index.ts

```diff
@@ -6,3 +6,7 @@ export {
 export { rehypeBetterReactMathjax } from './rehype-better-react-mathjax.js'
 export { rehypeExtractTocContent } from './rehype-extract-toc-content.js'
 export { rehypeIcon } from './rehype-icon.js'
+export {
+  twoslashRenderer,
+  rehypeTwoslashPopup
+} from './twoslash/rehype-twoslash-popup.js'
```

Comment:
can you remove this rehype plugin from the barrel `index.ts` file? because I start to receive warnings in console

```
docs:dev:  ⚠ ../node_modules/.pnpm/@typescript+vfs@1.6.0_typescript@5.6.3/node_modules/@typescript/vfs/dist/vfs.esm.js
docs:dev: Critical dependency: the request of a dependency is an expression
docs:dev: 
docs:dev: Import trace for requested module:
docs:dev: ../node_modules/.pnpm/@typescript+vfs@1.6.0_typescript@5.6.3/node_modules/@typescript/vfs/dist/vfs.esm.js
docs:dev: ../node_modules/.pnpm/twoslash@0.2.12_typescript@5.6.3/node_modules/twoslash/dist/index.mjs
docs:dev: ../node_modules/.pnpm/@shikijs+twoslash@1.20.0_typescript@5.6.3/node_modules/@shikijs/twoslash/dist/index.mjs
docs:dev: ../packages/nextra/dist/server/rehype-plugins/twoslash/rehype-twoslash-popup.js
docs:dev: ../packages/nextra/dist/server/rehype-plugins/index.js
docs:dev: ../packages/nextra/dist/server/compile.js
docs:dev: ../packages/nextra/dist/client/components/playground.js
docs:dev: ../packages/nextra/dist/client/components/index.js
docs:dev: 
docs:dev: ../node_modules/.pnpm/@typescript+vfs@1.6.0_typescript@5.6.3/node_modules/@typescript/vfs/dist/vfs.cjs.development.js
docs:dev: Critical dependency: the request of a dependency is an expression
docs:dev: 
docs:dev: Import trace for requested module:
docs:dev: ../node_modules/.pnpm/@typescript+vfs@1.6.0_typescript@5.6.3/node_modules/@typescript/vfs/dist/vfs.cjs.development.js
docs:dev: ../node_modules/.pnpm/@typescript+vfs@1.6.0_typescript@5.6.3/node_modules/@typescript/vfs/dist/index.js
docs:dev: ../node_modules/.pnpm/twoslash@0.2.12_typescript@5.6.3/node_modules/twoslash/dist/index.mjs
docs:dev: ../node_modules/.pnpm/@shikijs+twoslash@1.20.0_typescript@5.6.3/node_modules/@shikijs/twoslash/dist/index.mjs
docs:dev: ../packages/nextra/dist/server/rehype-plugins/twoslash/rehype-twoslash-popup.js
docs:dev: ../packages/nextra/dist/server/rehype-plugins/index.js
docs:dev: ../packages/nextra/dist/server/compile.js
docs:dev: ../packages/nextra/dist/client/components/playground.js
docs:dev: ../packages/nextra/dist/client/components/index.js
docs:dev: ../packages/nextra-theme-docs/dist/index.js
```

Reply:
I fixed it